src/Components/Checkbox.tsx

```diff
@@ -1,71 +1,114 @@
-import React from "react"
-
+import React, { Component, HTMLProps } from "react"
 import styled from "styled-components"
 import colors from "../Assets/Colors"
 
-interface Props extends React.HTMLProps<Checkbox> {
+export interface CheckboxState {
   checked: boolean
 }
 
-export class Checkbox extends React.Component<Props, null> {
-  static defaultProps = {
-    checked: true,
+export class Checkbox extends Component<HTMLProps<Checkbox>, CheckboxState> {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      checked: props.checked || false,
```

Comment:
I don't think we need `|| false` here?

Reply:
This is intentional so the `checked` will either be `true` or `false`, not `undefined` or `null`.